tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/actions/debugger_actions.ts

```diff
@@ -45,6 +45,12 @@ import {
  */
 export const debuggerLoaded = createAction('[Debugger] Debugger Loaded');
 
+export const debuggerUnloaded = createAction('[Debugger] Debugger Unloaded');
+
+export const debuggerDataPoll = createAction(
```

Comment:
I still think this should change to `debuggerDataPollStarted` or something like that.

Reply:
Renamed it to `debuggerDataPollOnset` for consistency.